README.md

```diff
@@ -21,35 +21,74 @@ This role requires root access, so either configure it in your inventory files,
 > playbook.yml:
 ```YAML
 - hosts: dnsserver
+  become: yes
   roles:
     - role: aalaesar.install_nextcloud
-      become: yes
 ```
 
 ## Role Variables
 
 Role's variables (and their default values):
 
-### Installation configuration
-> Source location will be calculated following channel, version and branch values.
+### Choose the version
```

Comment:
This is quite difficult to understand, maybe you could add various examples to elaborate for different scenarios?
Kind of like the Ansible documentation does it.